.github/workflows/pre-submit.cli.yml

```diff
@@ -1,5 +1,12 @@
-name: Pre submits
-on: [pull_request, workflow_dispatch]
+name: Pre submits cli
+on: 
+  pull_request:
+    branches: [ "main" ]
```

Comment:
I think we want these to run on `release/*` branches as well when we backport. I'm not sure about the `test/e2e` branch.

Reply:
done, thanks. I did not add the e2e test branch. Is it used?

Reply:
I'm not sure. We can leave it off for now.